src/components/EditPartnership/EditPartnership.tsx

```diff
@@ -10,11 +24,89 @@ interface EditPartnershipProps {
 
 export const EditPartnership: FC<EditPartnershipProps> = ({
   state,
-  // partnership,
+  partnership,
 }) => {
+  const [updatePartnership] = useUpdatePartnershipMutation();
+  const [deletePartnership] = useDeletePartnershipMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit = async (input: UpdatePartnershipInput) => {
```

Comment:
We should be careful conflating form state with other interfaces. For instance, `input.id` is valid to TS but it doesn't exist. I would be more comfortable with a separate form state interface that matches how the fields are defined and used here.

I wish there was a way to infer this automatically but given the nature of how the fields are defined it's impossible.

Reply:
Changed to use `UpdatePartnership` without ID